x-pack/plugins/cases/server/client/cases/update.test.ts

```diff
@@ -339,7 +367,337 @@ describe('update', () => {
+      clientArgs.services.caseService.patchCases.mockResolvedValue({
+        saved_objects: [{ ...mockCases[0] }],
+      });
+
+      await expect(
+        update(
+          {
+            cases: [
+              {
+                id: mockCases[0].id,
+                version: mockCases[0].version ?? '',
+                description: 'New updated description!!',
+              },
+            ],
+          },
+          clientArgs
+        )
+      ).resolves.not.toThrow();
+    });
+
+    it('does not update the description if the length is too long', async () => {
+      const description = Array(MAX_DESCRIPTION_LENGTH + 1)
+        .fill('a')
+        .toString();
+
+      await expect(
+        update(
+          {
+            cases: [
+              {
+                id: mockCases[0].id,
+                version: mockCases[0].version ?? '',
+                description,
+              },
+            ],
+          },
+          clientArgs
+        )
+      ).rejects.toThrow(
+        `Failed to update case, ids: [{"id":"mock-id-1","version":"WzAsMV0="}]: Error: The length of the description is too long. The maximum length is ${MAX_DESCRIPTION_LENGTH}.`
+      );
+    });
+
+    it('does not update the description if it is just an empty string', async () => {
+      await expect(
+        update(
+          {
+            cases: [
+              {
+                id: mockCases[0].id,
+                version: mockCases[0].version ?? '',
+                description: '',
+              },
+            ],
+          },
+          clientArgs
+        )
+      ).rejects.toThrow(
+        'Failed to update case, ids: [{"id":"mock-id-1","version":"WzAsMV0="}]: Error: The length of the description is too short. The minimum length is 1.'
+      );
+    });
+
+    it('does not update the description if it is a string with empty characters', async () => {
+      await expect(
+        update(
+          {
+            cases: [
+              {
+                id: mockCases[0].id,
+                version: mockCases[0].version ?? '',
+                description: '   ',
+              },
+            ],
+          },
+          clientArgs
+        )
+      ).rejects.toThrow(
+        'Failed to update case, ids: [{"id":"mock-id-1","version":"WzAsMV0="}]: Error: The length of the description is too short. The minimum length is 1.'
+      );
+    });
+  });
+
+  describe('Tags', () => {
+    const clientArgs = createCasesClientMockArgs();
+
+    beforeEach(() => {
+      jest.clearAllMocks();
+      clientArgs.services.caseService.getCases.mockResolvedValue({ saved_objects: mockCases });
+      clientArgs.services.caseService.getAllCaseComments.mockResolvedValue({
+        saved_objects: [],
+        total: 0,
+        per_page: 10,
+        page: 1,
+      });
+    });
+
+    it(`does not throw error when tags array is empty`, async () => {
```

Comment:
I wonder if we should throw for scenarios like this.

Calling `patch` without any params is allowed but we might as well notify the caller that something is probably wrong. @cnasikas ?